alpha/lib/enums/EntryServerNodeStatus.php

```diff
@@ -0,0 +1,14 @@
+<?php
+/**
+ * @package Core
+ * @subpackage model.enum
+ */
+interface EntryServerNodeStatus extends BaseEnum{
```

Comment:
This is the same as LiveEntryStatus so there is no reason to keep both ... choose one or the other and create the necessary client libs accordingly.